doc/overview/consensus/index.rst

```diff
@@ -29,55 +29,52 @@ By default CCF runs with CFT **and BFT is disabled on release versions**. To run
 Replica State Machine
 ---------------------
 
-Simplified
+Membership
 ~~~~~~~~~~
 
-Main states and transitions in CCF consensus. Note that while the implementation of the transitions differs between CFT and BFT, the states themselves do not.
+Any node of the network is always in one of four membership states. When using one-transaction reconfiguration, the ``Learner`` and 
+``RetirementInitiated`` states are not used and each node is either in the ``Active`` or ``Retired`` states. The dotted arrows in the 
+state diagram indicate a transition on rollback:
 
 .. mermaid::
 
-    graph LR;
-        Init-->Leader;
-        Init-->Follower;
-        Follower-->Candidate;
-        Candidate-->Follower;
-        Candidate-->Leader;
-        Leader-->Retired;
-        Follower-->Retired;
+    graph LR;  
+        Learner-->Active
+        Active-->RetirementInitiated
+        Active-->Retired
+        RetirementInitiated-.->Active
+        RetirementInitiated-->Retired
+        Retired-.->RetirementInitiated
+        Retired-.->Active
 
-Retirement details
-~~~~~~~~~~~~~~~~~~
+The membership state a node is currently is provided in the output of the :http:get:`/consensus` endpoint. 
 
-The transition towards retirement involves two additional elements of state:
+Simplified Leadership
+~~~~~~~~~~~~~~~~~~~~~
 
-- Retirement index (RI): Index at which node is set to ``Retired`` in ``public:ccf.gov.nodes.info``
-- Retirement Committable Index (RCI): Index at which the retirement transaction first becomes committable, ie. the first signature following the transaction.
-
-A node permanently transitions to ``Retired`` once it has observed commit reaching its Retirement Committable Index.
+Main consensus states and transitions. Note that while the implementation of the transitions differs between CFT and BFT, the states themselves do not.
+Nodes are not in any consensus state if they are not in the ``Active`` membership state yet, but once they are, they transition between all the 
+consensus states as the network evolves:
 
 .. mermaid::
 
-    graph LR;
-        Follower-->FRI[Follower w/ RI];
-        FRI-->Follower;
-        FRI-->FRCI[Follower w/ RCI];
-        FRCI-->Follower;
-        FRCI-->Retired;
+    graph LR;        
+        Candidate-->Follower;
```

Comment:
Leader to Candidate is not a valid transition. Follower to Follower and Candidate to Candidate are also valid transitions. See Raft paper:
![image](https://user-images.githubusercontent.com/42961061/143419490-737bad1f-d562-4bbe-8dd0-042ba95af33d.png)

Reply:
Great, thanks for the reference! We want exactly the same I think.